docs/try_local_system.md

```diff
@@ -338,6 +343,18 @@ Once you're done with configuring the application, proceed with
    2. Add the merchant ID you obtained in the previous step under the "current
       value" column for the `merchant_id` variable.
 
+#### Local Postman Variables 
+
+|variable|default|current|
+|:------:|:-----:|:-----:|
+|baseUrl||http://localhost:8080|
+|admin_api_key||test_admin|
+
+NOTE :- 
+To test stripe add stripe secret key to variable `connector_api_key`.
```

Comment:
I believe the `connector_api_key` variable is not a part of the hosted collection. Has it been included in the collection you updated?